README.rst

```diff
@@ -0,0 +1,184 @@
+nameko-statsd
+=============
+
+A StatsD dependency for `nameko <http://nameko.readthedocs.org>`_, enabling
+services to send stats.
+
+
+
+Usage
+-----
+
+To use the dependency you simply declare it on the service and then you
+can use it within any of the service methods (entrypoints, simple methods, etc.).
+
+
+.. code-block:: python
+
+    from nameko_statsd import StatsD, ServiceBase
+
+    class Service(ServiceBase):
+
+        statsd = StatsD('prod1')
```

Comment:
I'd say that it would be very common to only have one StatsD server available for our dependency. This will probably be the case on many scenarios.

This feature that allows you to select which one to select looks really useful, but I'd suggest we still allow the `StatsD` dependency to take no parameters for the case where we only provide one StatsD server in the Nameko config. This will simplify things and will make it simpler to use for that common case.